### PR TITLE
Adam_Berger_hw01

### PR DESCRIPTION
**Pull request by adamberger20**
Some reflection on my process is included in README.md. In general, I didn't run into too many problems once I realized that Markdown and R Markdown are not the same thing. I kept trying to follow the instructions for using R Markdown from the R for Data Science book. Once I realized that I needed to just follow the instructions on the "Mastering Markdown" guide, it was mostly smooth sailing. I ran into a few issues along the way:
1. Uploading the image: I didn't understand why the Preview wasn't working. I figured out eventually that I needed to upload the image to the repo, pull from the repo, and then preview it.
2. Pushing the local repo to GitHub: I followed the instructions on the "Using Git within RStudio" document, but I didn't understand why changes weren't showing up in GitHub. I did NOT figure this out; I just created a new local repository, copy-pasted my README.md, committed it to the local repo, and then pushed the local repo to GitHub. I thought this is what I had done originally, so I'm not sure what went wrong.
I liked the assignment, though! I'm sure this seems rudimentary to someone with any experience, but I had to be patient to trouble shoot some of these seemingly obvious issues. For the most part, that meant paying close attention to the "Mastering Markdown" guide and the cfss pages about using GitHub and general workflow. Although I did some googling to figure out why my files weren't pushing to GitHub, I didn't have any success troubleshooting that way.

### CODE REVIEW

**Review by msblanders — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

I think this was a  really solid first attempt -- Adam seems really capable, and as someone like him without much experience coding, it's nice to know there are smart people out there having as much trouble with this stuff as I am :)

A few issues I noticed were: possible spacing issues for the headers which didn't seem to appear properly for me. The image didn't render right for me, possibly because it wasn't uploaded to GitHub? Or maybe because the link doesn't require a '!' -- not sure. I found the mastering markdown link very helpful, so thank you! Otherwise, I thought everything was very well done and clear. Nice job!

**Comment by jwhenry28** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |    X       |              |            |
| **Ease of access**          |       X    |              |            |

Remarks:

This is a clean Readme. It is nice looking and easy to follow. I am sure that this kind of organization will translate well to future assignments when you are needing to explain your code in greater detail.

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+#*HW01 - Adam Berger *
+
+##Biography
```
Comment (zxgneil) [inline README.md:3]: You need to add a space between # and content, otherwise the headings will not show.

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+My name is Adam Berger and I am a fourth-year student in the College majoring in *Public Policy* and minoring in *Near Eastern Languages and Civilizations*. I am originally  from Bainbridge, Ohio, a small town about 45-minutes outside of Cleveland. On campus, I work as a tour guide in the Office of College Admissions and proudly serve as the RA of Alper House in Max Palevsky East.
+
+![Here's what I look like, featuring my Alper gear](IMG_2151.jpg)
```
Comment (zxgneil) [inline README.md]: I am not entirely sure why the picture is not showing. Your syntax should be correct. Maybe try simplifying image title/image name?

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |           |      x        |            |

Remarks:
1. The image and title does not render correctly. There needs to be a space between # and content. For the image, the file extension is case sensitive. You have "jpg" instead of "JPG" for you file, and thus github could not locate the file. Be careful next time!
1. I wish there will be more commit history for easier access.
1. Nice introduction and reflection.

File: README.md (hunk 2 above)
Comment (zxgneil) [inline README.md:7]: It should be "IMG_2151.JPG" because the file extension here is case sensitive.

**Pull request closed without merging**